modular_skyrat/modules/moretraitoritems/code/game/objects/items/clothing/suits/mafioso.dm

```diff
@@ -0,0 +1,17 @@
+/obj/item/clothing/suit/det_suit/noir/mafioso
+	name = "mafioso trenchcoat"
+	desc = "The guy wearing this means serious business..."
+	armor = list(MELEE = 25, BULLET = 20, LASER = 30, ENERGY = 30, BOMB = 30, BIO = 0, RAD = 10, FIRE = 30, ACID = 30, WOUND = 20)//Slightly worse than the sec coat
+
+/obj/item/clothing/under/suit/blacktwopiece/armoured
+	armor = list(MELEE = 10, BULLET = 10, LASER = 10,ENERGY = 10, BOMB = 0, BIO = 0, RAD = 10, FIRE = 50, ACID = 40)//Same as tacticool
+
+/obj/item/clothing/head/beret/sec/ocelot
+	name = "ocelot beret"
+	desc = "A red beret with the neo-russian insignia stamped on it. This bitch is wearing perfume."
+	armor = list(MELEE = 30, BULLET = 20, LASER = 20, ENERGY = 30, BOMB = 25, BIO = 0, RAD = 0, FIRE = 20, ACID = 50, WOUND = 4)
```

Comment:
Berets are refactored upstream by yours truly, so just keep that in mind, if you want it to look different.

Reply:
This was done pre mirror, and it takes the icon from the sec beret so it'll change when the overhaul is done.

Reply:
> This was done pre mirror, and it takes the icon from the sec beret so it'll change when the overhaul is done.

Yuh, I was just saying, if you need to know how to do it, just poke me on Discord.